spec/services/hyrax/custom_queries/navigators/find_files_spec.rb

```diff
@@ -1,19 +1,23 @@
-RSpec.describe Hyrax::CustomQueries::Navigators::FindFiles do
-  let(:query_service) { Valkyrie::MetadataAdapter.find(:test_adapter).query_service }
-  subject(:query_handler) { described_class.new(query_service: query_service) }
+RSpec.describe Hyrax::CustomQueries::Navigators::FindFiles, valkyrie_adapter: :test_adapter do
+  subject(:query_handler) do
+    described_class.new(query_service: Hyrax.query_service)
+  end
 
   describe '#find_files' do
     context 'when files exist' do
-      let!(:file_metadata1) { FactoryBot.create_using_test_adapter(:hyrax_file_metadata) }
-      let!(:file_metadata2) { FactoryBot.create_using_test_adapter(:hyrax_file_metadata) }
-      let!(:fileset) { FactoryBot.create_using_test_adapter(:hyrax_file_set, files: [file_metadata1, file_metadata2]) }
+      let(:file_metadata1) { FactoryBot.valkyrie_create(:hyrax_file_metadata) }
+      let(:file_metadata2) { FactoryBot.valkyrie_create(:hyrax_file_metadata) }
+      let(:fileset) { FactoryBot.valkyrie_create(:hyrax_file_set, files: [file_metadata1, file_metadata2]) }
+
       it 'returns file metadata resource' do
-        expect(query_handler.find_files(file_set: fileset).map(&:id).map(&:to_s)).to match_array [file_metadata1.id.to_s, file_metadata2.id.to_s]
+        expect(query_handler.find_files(file_set: fileset).map(&:id).map(&:to_s))
```

Comment:
Inefficient using chained maps, but I like how it reads.

Reply:
yeah, probably not what i would have done, but it's fine and not originating here.

Reply:
i think the `to_s` won't be needed much longer; since valkyrie deals with id<->string equality more robustly now